### DISCUSSION TITLE
curly/smart quotes?

### DISCUSSION THREAD

**Discussion by gommammi** [notaapp/talk#339] [Feature Requests] [answered]
Hi, I’m wondering if there’s an option to automatically transform the straight quote into a curly quote in Nota. The default straight quotes look a bit ugly in the text when I make a quotation of a word or a sentence, especially noticeable in serif fonts. (Of course, I can manually press option+[ and option+shift+[ for the curly ones whenever it appears, but it feels like an enormous burden.) Many text editors have this smart quote function (optional checkbox in the preference pane)—hope Nota will have this feature in the future as well. Many thanks!

**Comment by astoilkov** [comment, marked as answer]
Yeah, we have other users that request this as well. We should probably add this.
